src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,75 @@
+package duke.task;
+
+import duke.util.DukeException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Deadline extends Task {
+
+    private static final DateTimeFormatter DATE_SHORT_FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    private static final DateTimeFormatter DATE_MED_FORMATTER = DateTimeFormatter.ofLocalizedDate(FormatStyle.LONG);
```

Comment:
I like the idea of reusable date format, great idea to have. Perhaps to reduce from code duplicating, would you consider storing all the reused date formatter into a utility class instead ?